VictoriaMetrics/VictoriaMetrics
# vmui: fix step update on input blur in Firefox/Safari

Resolved a bug affecting step value updates in Firefox and Safari. Changes to step values failed to apply after clicking outside the input field.

To verify the fix:
1. Open the step input field.
2. Change the step value.
3. Click outside the input area to close the popup. The step on the chart should now reflect the changed value.
